pages/MarianaLoaiza/style.css

```diff
@@ -0,0 +1,98 @@
+
+body{
```

Comment:
No sure whether it meets BEM's requirement (only use classes).

Reply:
Hola @omeneses  en este no use BEM, ya que solo quiero que todo el body del html me tome las propiedades que le asigne al css.